### PR TITLE
RoutesDP

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

**Comment by GabrielAdward** [comment]
@albinforsbergju specific news and delete works with postman, I do not know how to check with PUT AND POST requests. Check if I have missed anything

File: backend/src/routes/person.js
Comment (albinforsbergju) [inline backend/src/routes/person.js:1]: Fix formatting in this file.

File: backend/src/routes/person.js [hunk 1]
```diff
@@ -34,6 +34,128 @@ router.get("/:id/news", async(req, res) => {
     }
 });
 
+router.get("/:id/news/:newsId", async (req, res) => {
```
Comment (albinforsbergju) [inline backend/src/routes/person.js:37]: What is the purpose of this route? We already have a route to get news connected to a person whom is a player or staff. 
If we only want one, we could create a news route and just call `/news/id`

File: backend/src/routes/person.js [hunk 2]
```diff
@@ -34,6 +34,128 @@ router.get("/:id/news", async(req, res) => {
...
+    res.status(500);
+    res.send({
+      errorCode: "internal_server_error",
+      errorMessage: "Failed to update news article"
+    });
+  }
+});
+
+
+router.delete("/:id/news/:newsId", async (req, res) => {
```
Comment (albinforsbergju) [inline backend/src/routes/person.js:130]: Same as above, should be in a news route and only `/news/id`

**Review by albinforsbergju — CHANGES_REQUESTED** [review]

File: backend/src/routes/person.js (hunk 1 above)
Comment (GabrielAdward) [reply backend/src/routes/person.js:37]: According to Jerome you cant use :id twice in same route. To wiew specific news

File: backend/src/routes/person.js (hunk 2 above)
Comment (GabrielAdward) [reply backend/src/routes/person.js:130]: According to Jerome you cant use :id twice in same route.

File: backend/src/routes/person.js (hunk 1 above)
Comment (GabrielAdward) [reply backend/src/routes/person.js:37]: the route should be club/:id/news/:newsId

**Review by albinforsbergju — APPROVED** [review]
LGTM

**Pull request merged**